tests/components/blebox/test_cover.py

```diff
@@ -365,17 +367,56 @@ async def test_with_no_stop(gatebox, hass, config):
     assert not supported_features & SUPPORT_STOP
 
 
-@pytest.mark.parametrize("wrapper", ALL_COVER_FIXTURES, indirect=["wrapper"])
-async def test_update_failure(wrapper, hass, config):
+@pytest.mark.parametrize("feature", ALL_COVER_FIXTURES, indirect=["feature"])
+async def test_update_failure(feature, hass, config, caplog):
     """Test that update failures are logged."""
 
-    feature_mock, entity_id = wrapper
+    caplog.set_level(logging.ERROR)
 
+    feature_mock, entity_id = feature
     feature_mock.async_update = AsyncMock(side_effect=blebox_uniapi.error.ClientError)
-    name = feature_mock.full_name
+    await async_setup_entity(hass, config, entity_id)
+
+    assert f"Updating '{feature_mock.full_name}' failed: " in caplog.text
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)